app.config.js

```diff
@@ -38,5 +38,9 @@ module.exports = {
   },
 
   // Wallets
-  portisId: process.env.GATSBY_PORTIS_ID || 'xxx'
+  portisId: process.env.GATSBY_PORTIS_ID || 'xxx',
+
+  // Toggle pricing option
```

Comment:
If we do not start documenting all the app options in the readme, then we should be at least more helpful in describing what those options do. Maybe like:

```tsx
// Pricing options: used to show or hide the fixed and dynamic price options to publishers during the price creation.
```

Reply:
@kremalicious  should we open this as another issue to document the app options? because I think is important to ensure future market owner aware of the options available. Might be a low priority now but in case the future we might have more options available.